pkg/installer/installer.go

```diff
@@ -84,9 +84,6 @@ func (i *installer) CreateUnconfiguredIgnition() (string, error) {
 	}
 
 	createCmd := fmt.Sprintf(templateUnconfiguredIgnitionBinary, openshiftInstallFilePath, i.EnvConfig.TempDir)
-	if swag.BoolValue(i.ApplianceConfig.Config.EnableInteractiveFlow) {
```

Comment:
Worth keeping it for backwards compatibility? i.e. for 4.20?

Reply:
How do we determine the ocp version from the OVE use case appliance-config.yaml? https://github.com/openshift/agent-installer-utils/blob/main/tools/iso_builder/config/4.20/appliance-config.yaml 

Maybe extract from the catalog string?

Reply:
based on @andfasano 's [comment](https://github.com/openshift/appliance/pull/548#discussion_r2477828821) above, we don't need a check isn't it?

Reply:
> based on @andfasano 's [comment](https://github.com/openshift/appliance/pull/548#discussion_r2477828821) above, we don't need a check isn't it?

yeah, it's probably indeed fine. we've only supported this flow for a single version (4.20) till now, so I guess it's not crucial.